src/app/services/baw-api/baw-api.service.ts

```diff
@@ -386,12 +460,16 @@ export class BawApiService<
     model: AbstractModel,
     options: BawServiceOptions = {}
   ): Observable<Model> {
-    const jsonData = model.getJsonAttributes?.({ update: true });
-    const formData = model.getFormDataOnlyAttributes({ update: true });
+    const jsonData = model.getJsonAttributesForUpdate();
     const body = model.kind
       ? { [model.kind]: jsonData ?? model }
       : jsonData ?? model;
 
+    let formData = model.getFormDataOnlyAttributesForUpdate();
+    if (options.params) {
+      formData = this.addUnscopedParams(formData, options.params);
+    }
```

Comment:
should probably also handle a json case (in which case just merge with body, or QSP is acceptable too)